CHANGELOG.md

```diff
@@ -1,5 +1,115 @@
 ### [Versions](https://material-ui.com/versions/)
 
+## 5.0.0-alpha.15
+
+###### _Nov 4 2020_
+
+Big thanks to the 20 contributors who made this release possible. Here are some highlights ✨:
+
+- ⚛️ Add support for React 17 (#23311) @eps1lon.
+  React 17 release is unusual because it doesn't add any new developer-facing features. It was released a couple of days ago. You can learn more about it in the [introduction post](https://reactjs.org/blog/2020/10/20/react-v17.html).
+- 🛠 Introduce a new `@material-ui/unstyled` package (#23270) @mnajdova.
+  The package will host the unstyled version of all the components. In this first iteration, only the Slider is available. You can find the component documented under the [same page](https://next.material-ui.com/components/slider-styled/#unstyled-slider) as the styled version.
+
+  **Why an unstyled package?**
+
+  While many teams are successfully building custom design systems wrapping Material-UI, we [often hear](https://github.com/mui-org/material-ui/issues/6218) that Material Design or our styling solution are something they don't need. Some teams prefer SASS, others prefer to customize the components starting from a pristine state. What all these teams have in common is that they value the features coming from the components, like the accessibility logic.
```

Comment:
```suggestion
  While many teams are successfully building custom design systems wrapping Material-UI, we [occasionally hear](https://github.com/mui-org/material-ui/issues/6218) that Material Design or our styling solution are something they don't need. Some teams prefer SASS, others prefer to customize the components starting from a pristine state. What all these teams have in common is that they value the features coming from the components, such as accessibility.
```

"Some teams prefer SASS, others prefer to customize the components starting from a pristine state."

This reads like these are two alternative choices.